php/ext/google/protobuf2/make-preload.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
```

Comment:
For distributing on pecl, should the bundle_php.c also be included?

Reply:
Yes. Where is the code that builds the pecl distribution?